cockatrice/src/dlg_filter_games.cpp

```diff
@@ -16,7 +16,13 @@
 DlgFilterGames::DlgFilterGames(const QMap<int, QString> &_allGameTypes,
                                const GamesProxyModel *_gamesProxyModel,
                                QWidget *parent)
-    : QDialog(parent), allGameTypes(_allGameTypes), gamesProxyModel(_gamesProxyModel)
+    : QDialog(parent), allGameTypes(_allGameTypes), gamesProxyModel(_gamesProxyModel),
+      gameAgeMap({{QTime(), tr("no limit")},
+                  {QTime(0, 5), tr("5 minutes")},
```

Comment:
I switched to not using the 5+ min because it looks really weird

Reply:
Agreed